src/app/core/models/common/advanced-settings.model.ts

```
@@ -69,7 +72,24 @@ export type AdvancedSettingValidatorRule = {
 
 export class AdvancedSettingsModel {
   static getDefaultMemoOptions(): string[] {
-    return ['employee_email', 'merchant', 'purpose', 'category', 'spent_on', 'report_number', 'expense_link'];
+    return ['employee_email', 'employee_name', 'merchant', 'purpose', 'category', 'spent_on', 'report_number', 'expense_link', 'card_number'];
+  }
+
+  static getMemoOptions(exportSettings: ExportSettingGet | NetSuiteExportSettingGet | QBOExportSettingGet, appName: string): string[] {
+    const defaultOptions = this.getDefaultMemoOptions();
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)